lib/osml-stacks/custom_model_endpoint.ts

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
What is the difference between this `custom_model_endpoint.ts` vs `test_model_endpoint.ts`

Reply:
Custom model endpoints only supports a user configuring their own model they wish to host with a SM endpoint - not one of our test models.